lib/features/tabs/ui/screens/components/category_items.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:elvan/shared/components/constants/app_colors.dart';
+import 'package:elvan/shared/components/constants/strings.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/src/widgets/container.dart';
+import 'package:flutter/src/widgets/framework.dart';
+
+class CategoryItems extends StatelessWidget {
+  const CategoryItems({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
```

Comment:
This Container should be a Generic component that behaves like a CARD with rounded corners and shape.